include/cadmium/celldevs/utils/grid_utils.hpp

```diff
@@ -0,0 +1,464 @@
+/**
```

Comment:
This file is huge, can it be break down into common areas of work or is all unrelated stuff just put together here?

Reply:
It is a little bit "cajón de sastre", to be honest... I will work on splitting it into different files

Reply:
This is fine to keep for now and refactor in a follow up PR.